services/movie.py

```diff
@@ -1,11 +1,13 @@
 from django.db.models import QuerySet
+from django.db import transaction
 
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
The type hint `list[int]` is used here, which is only supported in Python 3.9 and later. Ensure that your Python environment supports this syntax. If not, consider using `List[int]` from `typing` module for compatibility with earlier versions.